src/main/java/it/eng/idsa/businesslogic/service/impl/ProtocolValidationServiceImpl.java

```diff
@@ -0,0 +1,80 @@
+package it.eng.idsa.businesslogic.service.impl;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import it.eng.idsa.businesslogic.service.RejectionMessageService;
+import it.eng.idsa.businesslogic.util.RejectionMessageType;
+
+@Service
+public class ProtocolValidationServiceImpl {
```

Comment:
I was thinking whether it was needed since we only have that method. I could've made it a util class with the method being static, but static fields with @Value are not much recommended. So should I add an interface? :)